setup.py

```diff
@@ -10,7 +10,7 @@
 
 setup(
     name='hca-ingest',
-    version='2.2.0',
+    version='2.3.0',
```

Comment:
Do users of the ingest client need to change?
Some used a commit hash in the requirements file.

Reply:
I'm happy for us to rollout changes to consumers of the library as-and-when we need to make changes in projects that use the library.

So far it's only exporter that is even using the pip version of ingest-client rather than specific commit hashes pulled directly from GitHub